vnext/Microsoft.ReactNative/ReactInstanceSettings.idl

```diff
@@ -176,7 +176,13 @@ namespace Microsoft.ReactNative
       "If this is not provided, the value of @.JavaScriptBundleFile is used.")
     String DebugBundlePath { get; set; };
 
-    DOC_STRING("Base path used for the location of the bundle.")
+    DOC_STRING(
+      "Base path used for the location of the bundle. \n"
+      "This can be an `ms-appx://` or `ms-appdata://` URI (if the app is UWP or packaged using MSIX), "
+      "a filesystem path, or a URI pointing at an embedded resource.\n"
+      "To specify an embedded (`RCDATA`) resource, use the format `resource://moduleName` for @.BundleRootPath "
```

Comment:
Is resource:// something used anywhere else, or something we are making up?  It seems like a fairly generic name to be using.

Reply:
we have `ms-resource:///` although that is for resw files, so this is in the same vein although specific to this implementation.